test/srt/test_cache_report.py

```diff
@@ -0,0 +1,185 @@
+import asyncio
+import json
+import unittest
+
+import openai
+import requests
+
+from sglang.srt.utils import kill_child_process
+from sglang.test.test_utils import (
+    DEFAULT_MODEL_NAME_FOR_TEST,
+    DEFAULT_URL_FOR_TEST,
+    popen_launch_server,
+)
+
+
+class TestCacheReport(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.model = DEFAULT_MODEL_NAME_FOR_TEST
+        cls.base_url = DEFAULT_URL_FOR_TEST
+        cls.min_cached = 5
+        cls.process = popen_launch_server(
+            cls.model,
+            cls.base_url,
+            timeout=300,
+            other_args=[
+                "--chunked-prefill-size=40",
+                "--disable-cuda-graph",
```

Comment:
why do you disable cuda graph?

Reply:
I can remove it for better consistency with other tests. I run my test locally on an rtx2070m, so I need to disable cuda graph (and also change the model to gemma 2b) for the test to not go out of memory.